src/edu-benefits/schema.js

```diff
@@ -280,7 +280,7 @@ module.exports = {
           }
         }
       },
-      required: ['commissionYear', 'rotcScholarshipAmounts']
+      required: ['rotcScholarshipAmounts']
```

Comment:
Yep. `required` at each object's level only means that they're required at that level, and `seniorRotc` isn't set as required at the root level, so it can be left off.